src/renderer/src/components/IntlProvider/index.tsx

```diff
@@ -57,7 +60,9 @@ const getSupportedLocale = (locale: SupportedLanguageLocales): keyof typeof lang
 
 export const IntlProvider = ({ children }: { children: ReactNode }): JSX.Element | null => {
   const [appLocale, setAppLocale] = useState<AvailableLocales>()
+  const [minLoadTimePassed, setMinLoadTimePassed] = useState(false)
 
+  useTimeout(() => setMinLoadTimePassed(true), MIN_SPLASH_TIME)
```

Comment:
I don't think we should have a `minLoadTime`.  The splash screen should be declarative (aka based off state). In this case, the splash screen should be based off whether [setupApp](https://github.com/digidem/mapeo-desktop-shell/blob/56ba5eabfe8c30086e662f6bd224f8338ae20419/src/renderer/src/components/IntlProvider/index.tsx#L77) is complete.

Reply:
This is going to result in seeing the splash screen for 0.01s or less probably, basically a flash, is that what we want really?